### DISCUSSION TITLE
preference of splitting edge elevation for Edward Kort

### DISCUSSION THREAD

**Discussion by JeffBurris** [edwardkort/WWIDesigner#155] [NAF Study] [answered]
Hello, I was hoping to reach Mr. Kort, and of course anyone else is welcome to chime in with their thoughts. Please forgive me if he mentions in the material his preference, but I don't notice or remember it, off-hand.
I was wondering what his own personal preference was, as far as where to vertically place the splitting edge, such as 50% between flue ceiling and floor levels like Jon Norris, or all the way down at the bottom so the splitting edge is in line with the flue floor, like JP Gomez. 

It was interesting to note that Jon finds complexity and odd harmonics in the middle and clarity at the bottom, whereas others claim the exact opposite. I feel both camps have tested their theories, at least on the fly, and that both might be true based on other, differing factors between the flutes. Fipple factor has been a great leveler of the playing field of factors and countervaling factors that just can't, all be included as parameter, and it works pretty darned well. But I do need to address non-WID parameter factors for one voicing concern or so.

I place my splitting edge in the middle and the last thing I have is JP Gomez clarity, but I also don't do other things he does, from other TSH parameters to wall thickness. If anything, I'm looking to reduce just a bit of the buzz and, on the mic, white noise that is not just air blowing or a lack of complimentary bevels or chamfers between the bird and North TSH edge (I do that to attenuate hiss). It's reedy and woody in person but a mic can get a little saturated with odd harmonics or something and sound almost like white noise. But it's not just air hiss. It's complexity I'd like to attenuate a bit more, I'm not on the Holy Grail of Clarity quest; at some pointy it actually takes away from the in-person character of the voice, to me, to be too much like a recorder. A little "panflute nuance" (not too much) lends an air of seeming, traditional-esque authenticity to the voice, in my humble opinion. But there's too much for mic'ing up close, and I wouldn't mind controlling it even more.

If you're reading, Prof. Kort, where do you place your cutting edge? I'm not sure if 50% "elevation" is accurate in expressing where I put it, but it's halfway (vertically) between the bottom of the bird (flue ceiling) and the flue floor.
Thank you. 
Jeff Burris

**Comment by edwardkort** [comment, marked as answer]
Hi Jeff,
In the discussion that follows, keep these in mind:
* My preferred tone is dead clean and mellow
* I like very little back pressure - my splitting edge is quite sharp
* I incorporate a moveable splitting edge, so the length of the TSH is independent of modifications to the splitting edge shape.

For my geometry, the higher the splitting edge is in the air stream, the more mellow the lower notes. However, the lower the splitting edge is, the clearer the upper notes. So, I voice and tune a flute concurrently. I start with the lower notes and the splitting edge high. As I get to the upper notes, I lower the splitting edge enough that these high notes voice cleanly. Ultimately, this puts the splitting edge between a third to a half way down in the air stream.

Without the freedom of a moveable splitting edge, you will need to make a decision during the voicing process - before tuning. IMHO, you can't go wrong with the splitting edge in the middle of the air stream. It might not be the mellowest flute possible, but it will be close.

Cheers,
Edward Kort

**Reply by JeffBurris** [reply]
Thanks so much! I have had others weigh in and, that being the case, your input was quintessential to make a note of. I have been scared to prototype much anymore because it's income-related, now, and it's lost time and a hard but gratifying living. But I don't want to get entrenched in less of a pleasing voice than I can muster. I definitely think I will try voicing in this way, and not mill it so close to halfway down, to start off. The thought has been that, after I mill the splitting edge,  it becomes halfway down only after I just kiss-bevel the top side, by hand. But I can start milling higher with very little risk and give myself this better window. Thanks again.
